examples/with-markdown-shiki/astro.config.mjs

```diff
@@ -6,6 +6,9 @@
 // helpful tooltips, and warnings if your exported object is invalid.
 // You can disable this by removing "@ts-check" and `@type` comments below.
 import astroRemark from '@astrojs/markdown-remark';
+import fs from 'fs';
+
+const riGrammar = JSON.parse(fs.readFileSync('./src/shiki/ri.tmLanguage.json'));
```

Comment:
this feels like a really advanced case, I feel like most users just want to change the theme, and not add a custom lang. I'd recommend removing from the example. If you wanted to, you could maybe instead link to the docs with something like:

```diff
			{
				syntaxHighlight: 'shiki',
				shikiTheme: 'dracula',
+               // full documentation: ASTRO_DOCS_URL
```

Reply:
Yeah, that's a good appreciation

Reply:
I've made the changes. What do you think about them?